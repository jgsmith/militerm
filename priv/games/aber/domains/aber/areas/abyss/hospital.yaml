groups:
  control:
    npcs:
      beebop: 1
      rocksteady: 1
  web:
    npcs:
      blue: 1
      brown: 1
      green: 1
      pink: 1
      white: 1
locations:
  cell:
    npcs:
      guxx: 1
  control:
    groups:
      control: 1
  downstream:
    npcs:
      mucus: 1
  lolth:
    npcs:
      lolth: 1
  orcus:
    npcs:
      orcus: 1
  pain:
    npcs:
      death: 1
  pillars5:
    npcs:
      chuck: 1
  road1:
    npcs:
      braxx: 1
  secret:
    npcs:
      qbert: 1
  stairs:
    npcs:
      daemon: 1
  swamp3:
    npcs:
      mush: 1
  swamp6:
    npcs:
      jubilex: 1
  swamp7:
    npcs:
      ooze: 1
  web:
    groups:
      web: 1
  widow:
    npcs:
      widow: 1
npcs:
  beebop:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - beebop
          short: Beebop
          sight: >
            Beebop the Mutant Guard is standing here scratching his head.
      flag:
      identity:
        name: Beebop
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  blue:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - blue
          short: Blue
          sight: There is a blue spider here.
      flag:
      identity:
        name: Blue
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  braxx:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - braxx
          short: Braxx
          sight: >
            Smilin' Braxx the Salesdemon is here selling souvenirs and used weapons.
      flag:
      identity:
        name: Braxx
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  brown:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - brown
          short: Brown
          sight: There is a brown spider here.
      flag:
      identity:
        name: Brown
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  chuck:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - chuck
          short: Chuck
          sight: Chuck Gnoll is here.
      flag:
      identity:
        name: Chuck
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  daemon:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - daemon
          short: Daemon
          sight: Shadow Daemon is here skulking in the shadows.
      flag:
      identity:
        name: Daemon
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  death:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - death
          short: Death
          sight: The Death Knight stands ready to challenge you.
      flag:
        - no-summon
      identity:
        name: Death
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  green:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - green
          short: Green
          sight: There is a green spider here.
      flag:
      identity:
        name: Green
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  guxx:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - guxx
          short: Guxx
          sight: >
            The Grand Hoohah Guxx Unfufadoo is standing here testing his razor sharp claws.
      flag:
        - no-summon
        - no-exor
      identity:
        name: Guxx
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  jubilex:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - jubilex
          short: Jubilex
          sight: Jubilex the Faceless Lord is here.
      flag:
      identity:
        name: Jubilex
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  lolth:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - lolth
          short: Lolth
          sight: >
            Lolth, Demon Queen of the Spiders sits here waiting for her next victim.
      flag:
      identity:
        name: Lolth
        nominative: she
        objective: her
        possessive: her
    is_unique: true
  mucus:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - mucus
          short: Mucus
          sight: The mucus monster is here spitting phlegm at you.
      flag:
      identity:
        name: Mucus
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  mush:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - mush
          short: Mush
          sight: The brown mush sits here bubbling.
      flag:
      identity:
        name: Mush
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  ooze:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - ooze
          short: Ooze
          sight: A gray ooze slithers across the ground here.
      flag:
      identity:
        name: Ooze
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  orcus:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - orcus
          short: Orcus
          sight: The Demon Lord Orcus is here sizing up your skull.
      flag:
      identity:
        name: Orcus
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  pink:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - pink
          short: Pink
          sight: There is a pink spider here.
      flag:
      identity:
        name: Pink
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  qbert:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - qbert
          short: Qbert
          sight: >
            Qbert the Quasit is sitting at his desk going over the financial records.
      flag:
        - no-summon
      identity:
        name: Qbert
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  rocksteady:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - rocksteady
          short: Rocksteady
          sight: >
            Rocksteady the Mutant Guard is here slobbering on himself.
      flag:
      identity:
        name: Rocksteady
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  white:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - white
          short: White
          sight: There is a white spider here.
      flag:
      identity:
        name: White
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  widow:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - widow
          short: Widow
          sight: The Black Widow sits here spinning her web.
      flag:
      identity:
        name: Widow
        nominative: she
        objective: her
        possessive: her
    is_unique: true
