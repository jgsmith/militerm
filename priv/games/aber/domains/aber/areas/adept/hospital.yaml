groups:
  adept32:
    npcs:
      ramol: 1
      yinkit: 1
  adept49:
    npcs:
      anwar: 1
      hepquan: 1
  adept5:
    npcs:
      bear: 1
      beaver: 1
      bluejay: 1
      chipmunk: 1
      cricket: 1
      robin: 1
      woodchuck: 1
locations:
  adept10:
    npcs:
      baker: 1
  adept11:
    npcs:
      digger: 1
  adept14:
    npcs:
      teenager: 1
  adept18:
    npcs:
      peter: 1
  adept20:
    npcs:
      kelvin: 1
  adept21:
    npcs:
      blob: 1
  adept26:
    npcs:
      soldier: 1
  adept32:
    groups:
      adept32: 1
  adept34:
    npcs:
      fleeci: 1
  adept36:
    npcs:
      westfall: 1
  adept37:
    npcs:
      cropley: 1
  adept39:
    npcs:
      woodpecker: 1
  adept4:
    npcs:
      earthworm: 1
  adept41:
    npcs:
      executioner: 1
  adept47:
    npcs:
      termite: 1
  adept49:
    groups:
      adept49: 1
  adept5:
    groups:
      adept5: 1
  adept50:
    npcs:
      intendant: 1
  adept53:
    npcs:
      oldfart: 1
  adept57:
    npcs:
      dog: 1
  adept59:
    npcs:
      cactus: 1
  adept62:
    npcs:
      swabbie: 1
  adept66:
    npcs:
      mudclot: 1
  adept67:
    npcs:
      creeper: 1
  adept71:
    npcs:
      adept: 1
  adept72:
    npcs:
      captain: 1
  adept73:
    npcs:
      imp: 1
npcs:
  chipmunk:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - chipmunk
          short: Chipmunk
          sight: >
            A chipmunk chatterbox chitters crazily at the clouds.
      flag:
      identity:
        name: Chipmunk
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  dog:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - conrad
            - dog
          short: Conrad
          sight: >
            The accursed hound of the plains, Conrad the Gummy, hops around.
      flag:
      identity:
        name: Conrad
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  oldfart:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - oldfart
          short: Oldfart
          sight: >
            One of the oldfart trees leans over on a branch for support.
      flag:
      identity:
        name: Oldfart
        nominative: she
        objective: her
        possessive: her
    is_unique: true
  cropley:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - cropley
          short: Cropley
          sight: >
            Cropley is here in full bells and whistles, putting on a show for anyone who happens to be near
            and care even in the slightest.
      flag:
      identity:
        name: Cropley
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  yinkit:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - yinkit
          short: Yinkit
          sight: >
            The royal guard Yinkit stands rigid and stares impassively forward.
      flag:
        - no-zap
      identity:
        name: Yinkit
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  woodpecker:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - woody
            - woodpecker
          short: Woody
          sight: A vicious woodpecker flies at you, claws extended!
      flag:
      identity:
        name: Woody
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  cricket:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - cricket
          short: Cricket
          sight: >
            Crickets are known for their musical ability, especially when drunk.
      flag:
      identity:
        name: Cricket
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  earthworm:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - earthworm
          short: Earthworm
          sight: >
            A slow-moving piece of bait wriggles its earthwormy self toward you.
      flag:
        - no-hassle
        - no-magic
        - no-steal
      identity:
        name: Earthworm
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  creeper:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - creeper
          short: Creeper
          sight: >
            A long creeping underground vine snakes around your ankle.
      flag:
      identity:
        name: Creeper
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  cactus:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - cactus
          short: The Cactus
          sight: >
            The jaws of life on this elf-eating cactus gnash viciously!
      flag:
      identity:
        name: The Cactus
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  intendant:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - intendant
          short: Intendant
          sight: >
            She has been genetically altered. She is the Intendant.
      flag:
        - fireball
        - bhands
      identity:
        name: Intendant
        nominative: she
        objective: her
        possessive: her
    is_unique: true
  westfall:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - westfall
          short: Westfall
          sight: >
            The wizard of the castle, Westfall, attempts to make poisons even more deadly against the warring
            elves.
      flag:
        - icestorm
        - frost
        - shock
      identity:
        name: Westfall
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  bear:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - grizzly
            - bear
          short: Grizzly
          sight: A cuddly-looking grizzly prepares to cuddle you.
      flag:
      identity:
        name: Grizzly
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  kelvin:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - kelvin
          short: Kelvin
          sight: >
            Kelvin the arms dealer sells to all, human and elven alike.
      flag:
      identity:
        name: Kelvin
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  fleeci:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - fleeci
          short: Fleeci
          sight: >
            With her hubby away making an idiot himself on the field of battle, Queen Fleeci is being less
            than ladylike as she works on her martial arts skills. More than one guard has gone to barracks
            speaking soprano.
      flag:
      identity:
        name: Fleeci
        nominative: she
        objective: her
        possessive: her
    is_unique: true
  anwar:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - anwar
          short: Anwar
          sight: >
            Draxenguine Anwar of the main elven battalion is here.
      flag:
      identity:
        name: Anwar
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  captain:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - le capitan
            - captain
          short: Le Capitan
          sight: >
            The phantasm of Le Capitan is a direct result of &+LDEATH &*by scurvy!
      flag:
        - vtouch
      identity:
        name: Le Capitan
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  beaver:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - beaver
          short: Beaver
          sight: A beaver scuttles around, looking for food.
      flag:
      identity:
        name: Beaver
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  baker:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - baker
          short: Baker
          sight: >
            An enormously fat baker invites you in to his shop.
      flag:
      identity:
        name: Baker
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  hepquan:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - hepquan
          short: Hepquan
          sight: >
            Whistling while he works (aka sitting on his behind). Hepquan the elven norcor twirls his spear
            and generally makes a nuisance of himself.
      flag:
      identity:
        name: Hepquan
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  bluejay:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - bluejay
          short: Bluejay
          sight: A pretty bluejay chirps happily.
      flag:
      identity:
        name: Bluejay
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  imp:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - imp
          short: imp
          sight: >
            A thoroughly intoxicated imp is perched daintily on the harpsichord.
      flag:
        - no-hassle
        - no-steal
        - no-magic
      identity:
        name: imp
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  woodchuck:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - woodchuck
          short: Woodchuck
          sight: >
            This woodchuck couldn't chuck wood even if it wanted to.
      flag:
      identity:
        name: Woodchuck
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  swabbie:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - swabbie
          short: Swabbie
          sight: >
            A lowly deck swabbie attempts to keep the ship clean.
      flag:
      identity:
        name: Swabbie
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  robin:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - robin
          short: Robin
          sight: A pretty robin chirps happily.
      flag:
      identity:
        name: Robin
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  soldier:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - soldier
          short: Soldier
          sight: An insane soldier worries about his commander.
      flag:
        - no-hassle
        - no-steal
        - no-magic
      identity:
        name: Soldier
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  peter:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - peter
          short: Peter
          sight: >
            Peter drips sweat as he continuously pounds away at shoddy plate metal.
      flag:
      identity:
        name: Peter
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  executioner:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - executioner
          short: Executioner
          sight: >
            Wherever an elven soldier is found, the executioner's axe follows.
      flag:
        - no-steal
      identity:
        name: Executioner
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  blob:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - cytoplasmic blob
            - blob
          short: Cytoplasmic Blob
          sight: >
            Feasting on the garbage around the grate is a &+Rcytoplasmic blob&*.
      flag:
        - neg-fireball
        - neg-frost
        - neg-missile
        - neg-icestorm
      identity:
        name: Cytoplasmic Blob
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  mudclot:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - mudclot
          short: Mudclot
          sight: A blockage of mud clots the way down.
      flag:
      identity:
        name: Mudclot
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  teenager:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - teenager
          short: Teenager
          sight: >
            A really buff elven teenager is staring open-mouthed at the statue.
      flag:
        - frost
      identity:
        name: Teenager
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  adept:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - adept
          short: Adept
          sight: >
            This confused fossil of an elven cleric is an adept in many magical arts.
      flag:
        - no-hassle
        - no-magic
        - no-steal
      identity:
        name: Adept
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  termite:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - termite
          short: Termite
          sight: Oh wow! Look at the size of that termite!
      flag:
      identity:
        name: Termite
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  digger:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - digger
          short: Digger
          sight: >
            The spade-handling digger REALLY doesn't like you walking on his plants!
      flag:
      identity:
        name: Digger
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  ramol:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - ramol
          short: Ramol
          sight: >
            The retarded archer Ramol has been retired to less demanding duties than watching from the
            battlements. He looks up to Yinkit as a kind of brother as he wipes the frothing foam from his
            mouth.
      flag:
      identity:
        name: Ramol
        nominative: he
        objective: him
        possessive: his
    is_unique: true
