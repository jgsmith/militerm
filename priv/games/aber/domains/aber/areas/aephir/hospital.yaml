groups:
  aephir:
    npcs:
      aephir: 1
      runner: 1
locations:
  aephir:
    groups:
      aephir: 1
npcs:
  aephir:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - aephir
          short: Aephir
          sight: 
      flag:
        - no-punt
        - no-zap
        - no-summon
        - no-exorcise
        - no-alias
        - no-steal
        - no-burn
        - no-hassle
      identity:
        name: Aephir
        nominative: she
        objective: her
        possessive: her
    is_unique: true
  runner:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - elven runner
            - runner
          short: Elven Runner
          sight: 
      flag:
        - no-exorcise
        - no-zap
      identity:
        name: Elven Runner
        nominative: she
        objective: her
        possessive: her
    is_unique: true
