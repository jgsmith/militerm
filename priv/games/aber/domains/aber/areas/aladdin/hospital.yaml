groups:
  palace:
    npcs:
      jafar: 1
      princess: 1
      sultan: 1
  palace4:
    npcs:
      guard3: 1
      guard7: 1
  palace7:
    npcs:
      guard5: 1
      guard6: 1
locations:
  cave:
    npcs:
      abu: 1
  cliff:
    npcs:
      beast: 1
  lamp1:
    npcs:
      genie1: 1
  oasis3:
    npcs:
      snake: 1
  palace:
    groups:
      palace: 1
  palace11:
    npcs:
      guard8: 1
  palace12:
    npcs:
      guard9: 1
  palace2:
    npcs:
      guard1: 1
  palace3:
    npcs:
      guard2: 1
  palace4:
    groups:
      palace4: 1
  palace5:
    npcs:
      guard4: 1
  palace7:
    groups:
      palace7: 1
  palace8:
    npcs:
      iago: 1
  tower:
    npcs:
      genie2: 1
npcs:
  abu:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - abu
          short: Abu
          sight: There is a playful monkey swinging around here.
      flag:
        - no-hassle
      identity:
        name: Abu
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  beast:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - icy beast
            - beast
          short: The Icy Beast
          sight: >
            The Icy Beast roars at the sight of its new found dinner.
      flag:
      identity:
        name: The Icy Beast
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  genie1:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - genie
          short: Genie
          sight: An allpowerful, blue Genie floats here.
      flag:
        - no-hassle
      identity:
        name: Genie
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  genie2:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - genie
          short: Genie
          sight: >
            An all powerful, blue Genie floats here looking at you with hopeful eyes. He longs to be free and
            all you have to do is say it.
      flag:
        - no-hassle
      identity:
        name: Genie
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  guard1:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - abdul
            - guard
          short: Abdul
          sight: >
            Abdul is one of the more fashionably dressed castle guards.
      flag:
      identity:
        name: Abdul
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  guard2:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - ali
            - guard
          short: Ali
          sight: Ali, a castle guard, prepares to attack.
      flag:
      identity:
        name: Ali
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  guard3:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - mohammed
            - guard
          short: Mohammed
          sight: Mohammed, a castle guard, prepares to attack.
      flag:
      identity:
        name: Mohammed
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  guard4:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - hooman
            - guard
          short: Hooman
          sight: >
            Hooman the guard sharpens his sword looking forward to killing you.
      flag:
      identity:
        name: Hooman
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  guard5:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - mammad
            - guard
          short: Mammad
          sight: Mammad has been awaiting your arrival.
      flag:
      identity:
        name: Mammad
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  guard6:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - omid
            - guard
          short: Omid
          sight: Omid smiles, showing off his golden tooth.
      flag:
      identity:
        name: Omid
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  guard7:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - amon
            - guard
          short: Amon
          sight: Amon, a castle guard, prepares to attack.
      flag:
      identity:
        name: Amon
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  guard8:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - bijan
            - guard
          short: Bijan
          sight: >
            Bijan is really tired and really wants to kill you so he can take a nap.
      flag:
      identity:
        name: Bijan
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  guard9:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - ashkan
            - guard
          short: Ashkan
          sight: >
            Ashkan, the guard leader, is not pleased to see you.
      flag:
      identity:
        name: Ashkan
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  iago:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - iago
          short: Iago
          sight: >
            Iago screechs 'I will kill you if you give me another cracker!'
      flag:
      identity:
        name: Iago
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  jafar:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - jafar
          short: Jafar
          sight: >
            Jafar says, 'I dare you to challenge me... You street rat!'
      flag:
      identity:
        name: Jafar
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  princess:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - princess jasmine
            - princess
          short: Princess Jasmine
          sight: Princess Jasmine stares deeply into your eyes.
      flag:
        - no-hassle
      identity:
        name: Princess Jasmine
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  snake:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - slimey the snake
            - snake
          short: Slimey the Snake
          sight: >
            Slimey the Snake hung quietly from a tree... until you wandered in.
      flag:
      identity:
        name: Slimey the Snake
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  sultan:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - sultan
          short: Sultan
          sight: >
            The Sultan of Agrabah, Jasmine's father, is very pleased to see you.
      flag:
        - no-hassle
      identity:
        name: Sultan
        nominative: he
        objective: him
        possessive: his
    is_unique: true
