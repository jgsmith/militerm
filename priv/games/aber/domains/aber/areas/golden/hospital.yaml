groups:
  golden16:
    npcs:
      monkey: 1
      monkie: 1
      munkee: 1
  golden3:
    npcs:
      cockatoo: 1
      mawcaw: 1
locations:
  golden12:
    npcs:
      shaman: 1
  golden16:
    groups:
      golden16: 1
  golden17:
    npcs:
      venzual: 1
  golden19:
    npcs:
      arturo: 1
  golden26:
    npcs:
      xena: 1
  golden3:
    groups:
      golden3: 1
  golden37:
    npcs:
      statue: 1
  golden4:
    npcs:
      viper: 1
  golden8:
    npcs:
      panther: 1
  golden9:
    npcs:
      boar: 1
npcs:
  arturo:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - arturo
          short: Arturo
          sight: >
            Venzual's twin brother, Arturo, stands here looking for a wild boar to kill.
      flag:
      identity:
        name: Arturo
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  boar:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - wild boar
            - boar
          short: The Wild Boar
          sight: >
            A wild boar shuffles through the undergrowth looking for some insects to snack on.
      flag:
      identity:
        name: The Wild Boar
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  cockatoo:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - cockatoo
          short: Cockatoo
          sight: >
            A snow-white cockatoo is grooming itself along side the Mawcaw.
      flag:
        - no-steal
      identity:
        name: Cockatoo
        nominative: she
        objective: her
        possessive: her
    is_unique: true
  mawcaw:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - mawcaw
          short: Mawcaw
          sight: >
            A beautiful, multi-colored Mawcaw sits perched on a tree branch above you.
      flag:
        - no-steal
      identity:
        name: Mawcaw
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  monkey:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - monkey
          short: Monkey
          sight: >
            A small brown monkey is sitting on a vine, covering its eyes.
      flag:
        - blind
      identity:
        name: Monkey
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  monkie:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - monkie
          short: Monkie
          sight: >
            Another monkey is has hands firmly in place over its mouth.
      flag:
        - mute
      identity:
        name: Monkie
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  munkee:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - munkee
          short: Munkee
          sight: >
            There is a small brown monkey sitting on a vine, covering its ears.
      flag:
        - deaf
      identity:
        name: Munkee
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  panther:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - panther
          short: The Panther
          sight: >
            A sleek, black panther is alerted by your foot steps and prepares to pounce.
      flag:
      identity:
        name: The Panther
        nominative: she
        objective: her
        possessive: her
    is_unique: true
  shaman:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - shaman
          short: The Shaman
          sight: >
            The village Shaman bursts into a fiery rage as you intrude into his hut.
      flag:
        - fireball
        - shock
        - no-magic
      identity:
        name: The Shaman
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  statue:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - statue
          short: The Statue
          sight: >
            A large marble statue of a great emperor stands solemnly next to the temple entrance.
      flag:
      identity:
        name: The Statue
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  venzual:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - venzual
          short: Venzual
          sight: >
            A scrawny, cautious native watches you from behind a bush.
      flag:
      identity:
        name: Venzual
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  viper:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - viper
          short: The Viper
          sight: >
            A long, red, poisonous viper hisses at you and shows its fangs.
      flag:
      identity:
        name: The Viper
        nominative: he
        objective: him
        possessive: his
    is_unique: true
  xena:
    archetype: "aber:npc"
    data:
      detail:
        default:
          nouns:
            - xena
          short: Xena
          sight: >
            The legendary Xena, Guardian of the Mayan Temple, stands here ready to face any willing
            challenger.
      flag:
        - no-magic
      identity:
        name: Xena
        nominative: she
        objective: her
        possessive: her
    is_unique: true
